gatsby/src/components/link/link.tsx

```diff
@@ -3,7 +3,7 @@ import classnames from 'classnames';
 import { Link as TranslateLink } from 'gatsby-plugin-react-i18next';
 
 interface Props {
-  label: string;
+  label?: string;
```

Comment:
why?

Reply:
You can set content as children of component `<Link>... </Link>`